wikidataintegrator/wdi_helpers/publication.py

```diff
@@ -434,10 +447,37 @@ def europepmc_api_to_publication(ext_id, id_type):
     return p
 
 
+
+def arxiv_api_to_publication(ext_id, id_type='arxiv'):
+    """Make a Publication from an arXiv identifier."""
+    from manubot.cite.arxiv import get_arxiv_csl_item
```

Comment:
Important note - I had little interest in re-implementing @dhimmel and manubot's work here, so I'm co-opting his code :)

This means someone who wants to do this with arXiv identifiers should also `pip install manubot` first